lib/pipeline.js

```diff
@@ -50,6 +50,20 @@ class PipelineModel extends BaseModel {
     }
 
     /**
+     * Attach Screwdriver webhook to the pipeline's repository
+     * @method addWebhook
+     * @return {Promise}
+     */
+    addWebhook() {
+        return this.token.then(token =>
+            this.scm.addWebhook({
+                scmUri: this.scmUri,
+                token,
+                webhookUrl: 'https://api.screwdriver.cd/v4/webhooks'
```

Comment:
Do you suggest take in a parameter in this function and from the API pass in `request.server.info.uri`?